src/main/java/com/aiykr/iquais/exception/InvalidCredentials.java

```diff
@@ -0,0 +1,13 @@
+package com.aiykr.iquais.exception;
+
+public class InvalidCredentials extends RuntimeException{
```

Comment:
Rather than create individual exceptions lets create a custom exception class and then user ErrorConstants file to define error codes and their meanings